packages/apps/docs/src/components/Markdown/UnorderedList/styles.css.ts

```diff
@@ -0,0 +1,19 @@
+import { sprinkles, vars } from '@kadena/react-ui/theme';
+
+import { globalStyle, style } from '@vanilla-extract/css';
+
+export const ulListClass = style([
+  sprinkles({
+    marginY: '$5',
+    marginX: 0,
+    position: 'relative',
+  }),
+]);
+
+// sub-ul, won't work with classNames
+globalStyle('ul ul', {
```

Comment:
```suggestion
globalStyle(`${ulListClass} ul`, {
```

I think this works and it also makes styles only applied to this specific children of ul instead of affecting all `ul ul`

Reply:
how about the `li` marker styles, is it fixed separately?

Reply:
maybe something like this would work? ```
app.css.ts
import { style, globalStyle } from '@vanilla-extract/css';

export const parentClass = style({});

globalStyle(`${parentClass} > a`, {
  color: 'pink'
});```

Reply:
problem is, at least on my end, `globalStyle` isn't working with the classes defined, only with the html tags instead! It makes no sense, but that's what justifies the style choice here:


> I think this works and it also makes styles only applied to this specific children of ul instead of affecting all `ul ul`

Reply:
could you try `globalStyle(`${ulListClass} ul`, {...}`

Reply:
already did, and doesn't work! I've found a work-around to also solve the issue @realdreamer pointed out:
```typescript
// should work, but doesn't
// globalStyle(`${ulListClass} li > ul`, {
globalStyle(`article ul > li >ul`, {
  paddingLeft: vars.sizes.$4,
  margin: 0,
  top: `calc(-1 * ${vars.sizes.$4})`,
  position: 'relative',
});